docs/api/tasks-get-task-by-due-date.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+---
+
+# Get task by due date
+
+Returns an array of  [`task`](task) objects that contains only the task specified by the `due_date` parameter, if it exists.
+
+## URL
+
+```shell
+
+{server_url}/tasks/{due_date}
```

Comment:
```suggestion
{server_url}/tasks?due_date=<date_string to match>
```
Does this need a query parameter (like the suggestion) instead of  URL parameter?